KachingPlugIn/Controllers/KachingPlugInController.cs

```diff
@@ -67,8 +67,7 @@ public ActionResult UpdateProductImportUrl(string ProductsImportUrl)
             var configuration = Configuration.Instance();
             configuration.ProductsImportUrl = ProductsImportUrl;
             configuration.Save();
-            return RedirectToAction("Index");
-
+            return RedirectToAction("Index", "KachingPlugIn");
```

Comment:
This looks like something that belongs in the other PR. Just to keep the changes separated I would prefer if this is reverted for this PR.

Reply:
I guess a conflict sneaked in. Reverted now.